catalogs/management/commands/load_catalog.py

```diff
@@ -16,25 +19,54 @@ def add_arguments(self, parser):
         parser.add_argument("--name", type=str)
         parser.add_argument("--source", type=str, default=None)
         parser.add_argument("--catalog-version", type=str, default=None)
+        parser.add_argument("--load-standard-catalogs", action="store_true")
 
     def handle(self, *args, **options):
-        input_file = Path(options["catalog_file"])
+        if options["load_standard_catalogs"]:
+            self._load_standards()
+        else:
+            input_file = Path(options["catalog_file"])
+            name = input_name if (input_name := options["name"]) else input_file.name
+            create_kwargs = {}
+            for arg in ("source", "catalog_version", "impact_level"):
+                if value := options[arg]:
```

Comment:
Isn't `:=` an assignment operator not a comparison operator? So you are assigning the value of `options[arg]` to value, but does this `if` fail if `arg` doesn't exist?

Reply:
Order of operations: the assignment happens first, then the evaluation of the `if`, so effectively I'm checking whether or not `value` evaluates to `True` after it's been assigned. 

They're guaranteed to be in `options` since each has a default value.